lib/py/reddit_edgecontext/__init__.py

```diff
@@ -290,9 +290,13 @@ def loid(self) -> str:
             return self.loid_
 
         # Finally, return loid from authentication token
-        loid = self.authentication_token.loid
-        if loid:
-            return loid
+        try:
+            loid = self.authentication_token.loid
+            if loid:
+                return loid
+        except NoAuthenticationError:
```

Comment:
@spladug `NotImplementedError` should never happen here, correct?

Reply:
from the base AuthenticationToken class? it very much shouldn't and if it did that'd be a serious enough bug to warrant letting the exception through